Makefile

```diff
@@ -30,6 +30,6 @@ sweeper_test:
 	  exit 1; \
 	fi
 
-
-examples_to_docs:
+# This Makefile target runs notebooks listed below and converts them to markdown files in documentation/source/
+run_and_convert_notebooks_to_docs:
 	jupyter nbconvert --to markdown --output-dir="documentation/source/" --execute src/super_gradients/examples/model_export/models_export.ipynb
```

Comment:
There is no other notebook we want to add already ?

Reply:
Also, I cannot explain why but I got really confused by the name `run_and_convert_notebooks_to_docs`, it took me some time to understand what it means. Maybe my brain saw `run_and_convert` and thought about something like `notebook about running and converting a model`. 

So maybe we could rename it. A few ideas:
- `run_notebooks_and_export_as_md`
- `run_and_save_notebooks_as_md`

Reply:
All other notebooks in our repo are outdated 😢 & broken 😞 
And must be updated which I believe should be a separate effort (One PR per notebook)

It is probably a good idea to write some docs on how to write these notebooks and rethink the external data usage. In the nutshell:
* Notebooks MUST NOT require any user action to run them (User should not manually obtain API keys to download dataset, or authorize google drive access). Notebook must be self-contained, download datasets if necessary
* Unless demonstrating a feature that is hardware specific (CUDA AMP for example), notebook must not require a presence of GPU.
* Notebook runtime must not exceed 1h of execution time (The goal of notebooks to demonstrate concepts and not make users wait a day while long-term training finishes)